prime-router/src/main/kotlin/cli/OktaCommands.kt

```diff
@@ -195,6 +244,23 @@ abstract class OktaCommand(name: String, help: String) : CliktCommand(name = nam
     }
 
     companion object {
+        /**
+         * Base endpoint for the Okta account
+         */
+        lateinit var oktaBaseUrl: String
```

Comment:
Nit: having these as a lateinit vars made this a little harder to read.  One other possible way to do this would be to create a wrapper like `OktaConfig` and then collapse the `oktaBaseUrls` and `clientIds` maps into one

Reply:
Turned it into a wrapper, all good now.